src/components/wagonwheelComponent/legends/index.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react'; 
+import './style.css'; 
+
+class legends extends Component {
+    constructor (props){
+        super(props);
+        this.state = {}
+    }
+    render (){
+        return (
+            <div>
+                <ul className="legendWrapper">
```

Comment:
Follow consistent namings. Some places camel case("legendWrapper"), some places hyphen separated("each-legend"), somewhere else a mix of camel case and hyphen("App-title").